tensorboard/webapp/feature_flag/store/feature_flag_selectors_test.ts

```diff
@@ -53,4 +53,30 @@ describe('feature_flag_selectors', () => {
       expect(selectors.getIsInColab(state)).toEqual(false);
     });
   });
+
+  describe('#getIsGpuChartEnabled', () => {
+    it('returns value in the store', () => {
+      const state1 = buildState(
+        buildFeatureFlagState({
+          features: buildFeatureFlag({
+            enableGpuChart: false,
+          }),
+        })
+      );
+      const actual1 = selectors.getIsGpuChartEnabled(state1);
+
+      expect(actual1).toEqual(false);
```

Comment:
I don’t mind at all either way, but I thought that you liked using
`.toBe` here? Just curious whether there’s some nuance that I’m missing.

Reply:
Yup, that's possible. It was simply my oversight.